CONTRIBUTING.md

```diff
@@ -122,7 +122,7 @@ cmake --build .
 ```
 If you want to run tests also, generate build files using below command and then build.
 ```sh
-cmake -DBUILD_TESTING=ON -DBUILD_CURL_TRANSPORT=ON ..
+cmake -DBUILD_TESTING=ON -DBUILD_TRANSPORT_CURL=ON ..
```

Comment:
What if they don't specify any? Is it required to pick one?

Reply:
Not required.
Default transport depends on the OS where customer is building. Win-> WinHTTP,  Lin/Mac -> Curl

Reply:
In that case, remove that option from the getting started/contributing instructions to keep it simple.

Reply:
sure.  Will do it